### PR TITLE
Added default deployment-target for ios to 7.0.

### PR DESCRIPTION
**Pull request by maersu**
This PR defines a default Deployment Target for iOS devices in config.xml (See #3153 for more information).

*Testing*

1. Create a meteor App
1. Add iOS plattform
1. Build mobile packages
1. open XCode. The Deployment is 7.0 (or the one specified with `App.setPreference`)

With this Deployment Target and the following mobile-config.js it's possible to submit Apps to the Apple App store without errors and warnings.

```
App.icons({
  // iOS
  'iphone': 'resources/icons/icon-60x60.png',
  'iphone_2x': 'resources/icons/icon-60x60@2x.png',
  'iphone_3x': 'resources/icons/icon-180x180.png',
  'ipad': 'resources/icons/icon-76x76.png',
  'ipad_2x': 'resources/icons/icon-76x76@2x.png',
});
```

### CODE REVIEW

**Comment by brugnara** [comment]
+1

File: tools/commands-cordova.js [hunk 1]
```diff
@@ -1443,7 +1443,8 @@ var consumeControlFile = function (
   // set some defaults different from the Phonegap/Cordova defaults
   var additionalConfiguration = {
     'webviewbounce': false,
-    'DisallowOverscroll': true
+    'DisallowOverscroll': true,
+    'deployment-target': "7.0"
```
Comment (brugnara) [inline tools/commands-cordova.js]: For being compliant to the existent code, it should be `'7.0'` and not `"7.0"`. Am I wrong? :eyes:

Comment (maersu) [inline tools/commands-cordova.js]: @brugnara you are right i will change that

**Comment by Slava** [comment]
Merged in https://github.com/meteor/meteor/commit/c13bd1ec0a238f355125aef976fada8e320a189b. Thanks!

**Pull request closed without merging**